src/vs/workbench/parts/tasks/electron-browser/task.contribution.ts

```diff
@@ -610,6 +610,9 @@ class TaskService extends EventEmitter implements ITaskService {
 
 	private registerCommands(): void {
 		CommandsRegistry.registerCommand('workbench.action.tasks.runTask', (accessor, arg) => {
```

Comment:
No, it wouldn't. I could also add it to workbench.action.tasks.build too if necessary.